wolfcrypt/src/port/caam/caam_aes.c

```diff
@@ -120,8 +120,8 @@ int  wc_AesCbcEncrypt(Aes* aes, byte* out,
         word32 keySz;
         int ret;
 
-        if (wc_AesGetKeySize(aes, &keySz) != 0) {
-           return BAD_FUNC_ARG;
+        if ((ret = wc_AesGetKeySize(aes, &keySz)) != 0) {
```

Comment:
Lets try to keep the number of changes in caam_aes.c to a minimum. If we can avoid these changes to the checks already in place then lets please do that.

Reply:
`wc_AesGetKeySize()` only returns `0` or `BAD_FUNC_ARG`, so there's no change in behavior.  All the other ports (and the software version) percolate the error from `wc_AesGetKeySize()`, so the changes I made in caam_aes.c just harmonize CAAM with that.

The `FREESCALE_LTC` port wasn't even checking the `wc_AesGetKeySize()` retval for failure at all... I fixed that with my last commit, and now all the implementations are consistent with each other.